tests/fixtures/vue-bind/binding-with-template-string.vue

```diff
@@ -4,6 +4,9 @@
       Hello World
     </div>
     <div :class="`${isTrue ? 'a' : 'b'}`"></div>
+    <div :style="'display: none !important'">
```

Comment:
This is not a template string anymore, can you also test that the template string binding keeps on working?

Reply:
I moved it to binding.vue